src/GPUArrays.jl

```diff
@@ -9,6 +9,7 @@ using LinearAlgebra.BLAS
 using Base.Cartesian
 
 using Adapt
+using LLVM
```

Comment:
I typically add a `using LLVM.Interop` after that so that you don't have to specify the modules for `assume`.

Reply:
Sounds good~